tests/unit/test_charm.py

```diff
@@ -441,6 +441,20 @@ def test_rewrite_enabled_property(self):
         conf_or_rel = self.harness.charm._all_config_or_relations[0]
         self.assertEqual(conf_or_rel._rewrite_enabled, False)
 
+    def test_whitelist_source_range(self):
+        self.harness.update_config({"whitelist-source-range": "10.0.0.0/24,172.10.0.1"})
+        conf_or_rel = self.harness.charm._all_config_or_relations[0]
+        self.assertEqual(conf_or_rel._whitelist_source_range, "10.0.0.0/24,172.10.0.1")
+        result_dict = conf_or_rel._get_k8s_ingress().to_dict()
+        # Expect default values plus the one we've added.
+        expected = {
```

Comment:
I wouldn't compare the rest of the properties values in case they are changes. It'll make the tests hard to maintain